services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/StaticAttribute.java

```diff
@@ -216,6 +227,15 @@ private Builder(EnhancedType<R> type) {
             return this;
         }
 
+        /**
+         * An {@link AttributeConverter} for the attribute type ({@link EnhancedType}), that can convert this attribute.
+         * It overrides any converter for this type provided by the table schema {@link AttributeConverterProvider}.
```

Comment:
Nit: Overrides -> takes precedence over. It doesn't change the table schema.

Reply:
Yes